sorc/ncep_post.fd/SURFCE.f

```diff
@@ -2609,9 +2610,33 @@ SUBROUTINE SURFCE
 !
 ! SRD
 !
-!        ANEMOMETER LEVEL (10 M) MAX WIND SPEED.
+!        ANEMOMETER LEVEL (10 M) MAX WINDS.
 !
-      IF (IGET(422)>0) THEN
+      IF ((IGET(422)>0) .OR. (IGET(783)>0) .OR. (IGET(784)>0)) THEN
```

Comment:
@KarinaAsmar-NOAA Two setting options should be applied for max wind speed only. You may remove "IGET(783)>0" and "IGET(784)>0".